libfwupd/fwupd-enums.h

```diff
@@ -241,12 +241,13 @@ typedef enum {
  * @FWUPD_VERSION_FORMAT_NUMBER:		A single integer version number
  * @FWUPD_VERSION_FORMAT_PAIR:			Two AABB.CCDD version numbers
  * @FWUPD_VERSION_FORMAT_TRIPLET:		Microsoft-style AA.BB.CCDD version numbers
- * @FWUPD_VERSION_FORMAT_QUAD:			Dell-style AA.BB.CC.DD version numbers
+ * @FWUPD_VERSION_FORMAT_QUAD:			UEFI-style AA.BB.CC.DD version numbers
  * @FWUPD_VERSION_FORMAT_BCD:			Binary coded decimal notation
  * @FWUPD_VERSION_FORMAT_INTEL_ME:		Intel ME-style bitshifted notation
  * @FWUPD_VERSION_FORMAT_INTEL_ME2:		Intel ME-style A.B.CC.DDDD notation notation
  * @FWUPD_VERSION_FORMAT_SURFACE_LEGACY:	Legacy Microsoft Surface 10b.12b.10b
  * @FWUPD_VERSION_FORMAT_SURFACE:		Microsoft Surface 8b.16b.8b
+ * @FWUPD_VERSION_FORMAT_DELL_ESRT:		Dell ESRT BB.CC.DD style
```

Comment:
I think I would prefer this to be `DELL_BIOS` or `DELL_SYSTEM` as it's only valid for use with the system firmware.  Device firmware doesn't have the same definition necessarily.

Reply:
"system" is a really hand-wavey noun -- I think "bios" is much better understood in this context. I'll switch everything to `dell-bios` and then work on the LVFS side.